docs/workflows/transitions/postprocess/postprocess/utils.py

```diff
@@ -35,3 +35,16 @@ def convert_predictions_to_v2(predictions):
             result[label] = sorted(result[label], key=lambda p: -p['confidence'])
 
     return result
+
+
+def to_validated_format(predictions):
+    def add_metainfo(el):
+        return {'isEdited': False, 'automated': True, **el}
```

Comment:
Could we give the argument here a more suitable name?
```suggestion
    def add_metainfo(field_prediction_v2):
        return {'isEdited': False, 'automated': True, **field_prediction_v2}
```